1436790/1438462: Part 11: 17 files with MSDN link fixes. (#10330)
dotnet/docs: docs/framework/data/adonet/sql/linq/index.md

 Provides links to Visual Basic and C# samples.  
  
## Related Sections  

>  [Language-Integrated Query (LINQ) - C#](../../../../../csharp/programming-guide/concepts/linq/index.md)

Reviewer: that backslash was necessary to break the line
![image](https://user-images.githubusercontent.com/12971179/52390389-e5a22280-2a4c-11e9-9ebc-eb97b85ad8ff.png)

 Provides links to Visual Basic and C# samples.  
  
## Related Sections  

>  [Language-Integrated Query (LINQ) - C#](../../../csharp/programming-guide/concepts/linq/index.md)  

Reviewer: You have this and several other broken links in the build report. Take a look. I'd also break the links in two where you have the C# and VB version. 

```suggestion
 [Language-Integrated Query (LINQ) - C#](../../../../../csharp/programming-guide/concepts/linq/index.md)\
 Provides overviews of LINQ technologies in C#.
 
```

Author: @mairaw My LINQ-C# and LINQ-VB links (going up 3 parent folders instead of 5) work in VS Code preview. I also don't understand how you can go up 5 folder levels, since 3 gets you to a top-level folder in the repo. Does using 5 lead to another repo with the same file structure?

Reviewer: I just counted how many subfolders you had in here and took a guess on why these links are broken:
The file is docs/framework/data/adonet/sql/linq/index.md

Going up just 3 levels brings you to the data folder.

>  [Language-Integrated Query (LINQ) - Visual Basic](../../../visual-basic/programming-guide/concepts/linq/index.md)  

Reviewer: ```suggestion
 [Language-Integrated Query (LINQ) - Visual Basic](../../../../../visual-basic/programming-guide/concepts/linq/index.md)  
```

>  Provides overviews of [!INCLUDE[vbteclinq](../../../../../../includes/vbteclinq-md.md)] technologies.  

Reviewer: ```suggestion
 Provides overviews of LINQ technologies in Visual Basic.
```
